models/issue.go

```diff
@@ -1785,3 +1786,13 @@ func UpdateIssuesMigrationsByType(gitServiceType structs.GitServiceType, origina
 		})
 	return err
 }
+
+// RefEndName returns the end name of an issue's ref.
+func (issue *Issue) RefEndName() string {
```

Comment:
Please don't reference git module from models. It's better to put that on `services/issue` as a method.

Reply:
@lafriks what do you think?